back/src/services/messageService.js

```diff
@@ -0,0 +1,61 @@
+/* eslint-disable quotes */
+import { Message } from "../models/Message.js";
+
+async function createMessage({ authorName, userId, roomId, text }) {
+  try {
+    const result = await Message.create({
+      authorName,
+      userId,
+      roomId,
+      text,
+    });
+
+    return result;
+  } catch {
+    return {};
+  }
+}
+
+function getById(id) {
+  return Message.findOne({
+    where: { id },
+  });
+}
```

Comment:
it seems like u r not handling error cases on all the message routes, when the the message with the provided id/roomId doesn't exist
pls fix this everywhere, also same for `user` and `room` files, handle cases when nothing is found in db by provided ids

Reply:
As for messages, this function is not really in use in this app. As for others maybe I don't understand it right, but all other cases are handled by catchError middleware and sequelize. If it is not enough I'll do additional handling.

Reply:
Ah, sorry, didn't mention catchError wrapper for controllers, then it should work fine)